docs/docker-image.rst

```diff
@@ -15,7 +15,6 @@ Relevant images are:
 
 - ``lsstsqre/ltd-keeper:latest`` --- Created by the Dockerfile in this `github.com/lsst-sqre/ltd-keeper <https://github.com/lsst-sqre/ltd-keeper>`_ repository. This deploys LTD Keeper as a uWSGI application.
 - ``lsstsqre/nginx-python:compose`` --- An Nginx Proxy for LTD Keeper containers to be used in a Docker Compose environment (see :doc:`compose`). The related Dockerfile can be found in `github.com/lsst-sqre/nginx-python-docker <https://github.com/lsst-sqre/nginx-python-docker>`_.
-- ``lsstsqre/nginx-python:k8s`` --- An Nginx Proxy for LTD Keeper containers to be used in a Kubernetes environment (see :doc:`gke-deploy`). The related Dockerfile can be found in `github.com/lsst-sqre/nginx-python-docker <https://github.com/lsst-sqre/nginx-python-docker>`_.
```

Comment:
Ended up using a pre-made image from Google Cloud Platform that's configured to do TLS termination, rather than making my own.